src/storage/query/QueryBaseProcessor.inl

```diff
@@ -517,7 +517,7 @@ kvstore::ResultCode QueryBaseProcessor<REQ, RESP>::collectEdgeProps(
         lastRank = rank;
         lastDstId = dstId;
         RowReader reader = RowReader::getEmptyRowReader();
-        if ((!onlyStructure || retTTL.has_value()) && !val.empty()) {
+        if ((!onlyStructure || exp_ != nullptr|| retTTL.has_value()) && !val.empty()) {
```

Comment:
What about the code as below?
```
if (!onlyStructure && !val.empty()) {
}
```

Reply:
if only get dst, the `onlyStructure` is true, the filter is no use.

Reply:
> if only get dst, the `onlyStructure` is true, the filter is no use.

Got.